src/config/lwwMap.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { entriesOf } from '@salesforce/ts-types';
+import { nowBigInt } from '../util/time';
+import { LWWRegister } from './lwwRegister';
+import { ConfigContents, Key } from './configStackTypes';
+
+export const SYMBOL_FOR_DELETED = 'UNIQUE_IDENTIFIER_FOR_DELETED' as const;
+
+export type LWWState<P> = {
+  [Property in keyof P]: LWWRegister<P[Property] | typeof SYMBOL_FOR_DELETED>['state'];
+};
+
+/**
```

Comment:
Might be nice to have a definition for LWW and CRDT in at least one of these jsdocs - just thinking about the developer who's trying to fix a bug in here 5 years from now

Reply:
yeah, I did it at the Register level.  

https://github.com/forcedotcom/sfdx-core/blob/b0f12e2e11d9d77e4c3ffecdd5ca32017bb53705/src/config/lwwRegister.ts#L17-L18

Would you want it on the LWWMapp, too?

Reply:
sfdx-core has generated docs from code so I think it should be added here.

Reply:
it's not exported so it shouldn't be in the public docs, right?